### PR TITLE
[RTM] Use the file meta data by default when adding an image

### PR DESCRIPTION
**Pull request by leofeyer**
This PR implements contao/core#7774. It is still work in progress, but it already works for adding images to text elements.

### CODE REVIEW

**Comment by leofeyer** [comment]
Default form:



Form with the "Overwrite meta data" option set:

**Comment by leofeyer** [comment]
This PR is ready to merge now.

File: src/Resources/contao/classes/Backend.php [hunk 1]
```diff
@@ -714,9 +714,13 @@ public static function findSearchablePages($pid=0, $domain='', $blnIsSitemap=fal
 	 * @param string  $strUuid
 	 * @param string  $strPtable
 	 * @param integer $intPid
+	 *
+	 * @deprecated Deprecated since Contao 4.4, to be removed in Contao 5.0.
 	 */
 	public static function addFileMetaInformationToRequest($strUuid, $strPtable, $intPid)
 	{
+		@trigger_error('Using Backend::addFileMetaInformationToRequest() has been deprecated and will no longer work in Contao 5.0.', E_USER_DEPRECATED);
```
Comment (Toflar) [inline src/Resources/contao/classes/Backend.php]: Use what instead?

Comment (leofeyer) [inline src/Resources/contao/classes/Backend.php]: Nothing. The elements now directly use the meta data, so we do not need this workaround anymore.

Comment (Toflar) [inline src/Resources/contao/classes/Backend.php]: Ok :)

**Pull request merged**